api/src/main/resources/api/service_openapi.yaml

```diff
@@ -876,3 +941,43 @@ components:
         - id
         - title
       additionalProperties: true
+
+    #########################UNDERLAY CONFIG V2 [START]#################################
+    UnderlayV2:
+      type: object
+      description: Underlay
+      properties:
+        name:
+          description: Name
+          type: string
+        description:
+          description: Description
+          type: string
+        uiConfiguration:
+          description: UI configuration
+          type: string
```

Comment:
Can this be handled as generic object instead? The string was unavoidable when it had to pass through protos without actually being in the definition but it has significant downsides in that it can't be automatically formatted along with the rest of the underlay and errors aren't caught until it's actually parsed instead of at load time.

Reply:
Do you mean if we could define a schema here for the UI config as returned by the API? Yes, and that sounds like a good idea. In that case then, I think I'm going to just remove this for now until we have our discussion about what the UI config should look like.

Reply:
I had just meant a `map<string, Object>` or similar using `additionalProperties: true` but defining a schema would be fine too.